### PR TITLE
automated application acknowledgement E-Mail

### PR DESCRIPTION
**Pull request by fseipel**

### CODE REVIEW

File: Classes/Controller/Application/SubmitController.php [hunk 1]
```diff
@@ -37,12 +43,60 @@ public function showSummaryAction(Application $application)
...
+        } else {
+            $this->redirect("submited", null, null, ['application' => $application]);
+        }
+    }
+
+    /**
+     * @param  Application $application
+     * @return void
+     */
+    public function submitedAction(Application $application)
```
Comment (pm-sschreiber) [inline Classes/Controller/Application/SubmitController.php]: Typo in action name, "submited" should be "submitted".

Comment (pm-sschreiber) [reply Classes/Controller/Application/SubmitController.php]: Why are the two additional methods necessary? Couldn't this also be done in the SubmitAction (while moving the mail stuff into a service function to slim down controller code) and/or in a separate, non-action function in the controller?

Comment (fseipel) [reply Classes/Controller/Application/SubmitController.php]: If everything is done in the submit action the email/application would be resend on page reload.
I am going to move the e-mail stuff into the submit action(Service/Function) as it makes sense, but keep the submitted action.

**Comment by pm-sschreiber** [comment]
@fseipel 

I have fixed two issues for you:

- config setting names were completely different in ext_conf_template.txt. Therefore it was not possible to activate those settings via EM.
- I moved the initial feature check (whether to send auto ack. emails) back into the controller so no message object is built when the feature is not activated.

**Pull request merged**